extensions/stackdriver/metric/record.cc

```diff
@@ -33,6 +38,34 @@ void record(bool is_outbound, const ::wasm::common::NodeInfo &local_node_info,
       request_info.request_protocol == ::Wasm::Common::kProtocolGRPC
           ? request_info.request_url_path
           : request_info.request_operation;
+
+  const auto &local_labels = local_node_info.labels();
+  const auto &peer_labels = peer_node_info.labels();
+
+  std::string local_canonical_name = local_node_info.workload_name();
```

Comment:
How about this?
```
const auto local_name_iter = local_labels.find(kCanonicalNameLabel);
const string& local_canonical_name = local_name_iter == local_labels.end() ? 
       local_node_info.workload_name() : local_name->second;
```

I think this should compile and save an allocation.

Reply:
done.